src/main/java/com/github/algobot76/surabaya/util/StaticAnalysisVisitor.java

```diff
@@ -34,7 +36,8 @@ public void visit(ClassOrInterfaceDeclaration n, Project project) {
 			Range endRange = tokenRange.getEnd().getRange().get();
 			lineCount = endRange.begin.line - beginRange.begin.line;
 		}
-		Class newClass = new Class(n.getName().toString(), type, n.getModifiers().get(0).toString().trim(), lineCount);
+		System.out.printf("Parsing %s\n", n.getName());
```

Comment:
are Abstract Classes handled?

Reply:
Oh nice catch, totally forgot about that 😛

Reply:
Added now